packages/playground/website/public/logger.php

```diff
@@ -59,6 +59,16 @@ function response($ok, $error = null)
     }
     $text .= "\n\nUrl\n\n" . $_POST['url'];
 }
+
+if (isset($_POST['context']) && !empty($_POST['context'])) {
+    $text .= "\n\nContext\n\n" . print_r($_POST['context'], true);
```

Comment:
would var_export be useful to distinguish between null, false, empty strings? or not really?

Reply:
I don't think it would add value because our code adds the data, but I can see how this would be useful for user-defined data.

Reply:
`print_r` isn't needed here, I will remove it.